src/ctapipe/core/tool.py

```diff
@@ -282,9 +282,16 @@ def load_config_file(self, path: str | pathlib.Path) -> None:
             with open(path, "rb") as infile:
                 config = Config(toml.load(infile))
             self.update_config(config)
-        else:
```

Comment:
Okay, so the issue is _traitlets_ silently ignores errors in `load_config_file`? That would be worth an upstream issue I guess.

The intended behavior here was that we pass any format not directly implemented by us to traitlets, so we get all traitlets formats without having to know what is supported.

Reply:
Yes, good fix for now. Maybe we should follow up with an issue upstream

Reply:
Looking at traitlets, the situation is even worse.

Setting `--config=example.conf` will result in loading `example.json` and `example.py` if they are in the same directory. TRaitlets splits off the extension, never looks at it and then  loads all files that have the same stem and one of the supported extensions (json, py)

Reply:
I opened an issue about this 1.5 years ago: https://github.com/ipython/traitlets/issues/850